packages/typescript/internal/ts_project.bzl

```diff
@@ -128,6 +143,7 @@ def _ts_project_impl(ctx):
             arguments = [arguments],
             outputs = outputs,
             executable = "tsc",
+            execution_requirements = execution_requirements,
```

Comment:
I would expect we need a mnemonic here to key the worker pool?

Reply:
done